lib/src/simple_op_heads_store.rs

```diff
@@ -236,3 +254,65 @@ impl OpHeadsStore for SimpleOpHeadsStore {
         })
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::collections::HashSet;
+    use std::fs;
+    use std::path::Path;
+
+    use itertools::Itertools;
+
+    use super::InnerSimpleOpHeadsStore;
+    use crate::op_heads_store::OpHeadsStore;
+    use crate::op_store::OperationId;
+    use crate::simple_op_heads_store::SimpleOpHeadsStore;
+
+    fn read_dir(dir: &Path) -> Vec<String> {
+        fs::read_dir(dir)
+            .unwrap()
+            .map(|entry| entry.unwrap().file_name().to_str().unwrap().to_string())
+            .sorted()
+            .collect()
+    }
+
+    #[test]
+    fn test_simple_op_heads_store_migration() {
```

Comment:
Nice. Thanks for testing the migration. That's better than I've ever done (only manual tests for me) :)

Reply:
Ack :)